### ISSUE TITLE
bug: icon name in the desktop entry is empty

### ISSUE THREAD

**Issue by wiiznokes** [cosmic-utils/web-apps#63]
To reproduce:
create a desktop entry for `https://github.com`

Those icons will be available: (only two are shown in the UI)

```
[src/favicon.rs:44:5] &favicons = [
    "https://github.com/fluidicon.png",
    "https://github.githubassets.com/assets/pinned-octocat-093da3e6fa40.svg",
    "https://github.githubassets.com/favicons/favicon.png",
    "https://github.githubassets.com/favicons/favicon.svg",
]
```
Create the entry

The Icon field will be `Icon=/home/fedasus/.local/share/icons/QuickWebApps/.svg`

**Comment by elevenhsoft** [comment]
Oh, it should happen if you type URL and fetch favicon before typing the name of Web App. I did not consider this case so that's happening right now ^^ Gonna fix it.

**Comment by elevenhsoft** [comment]
It should be fixed now with this commit: https://github.com/cosmic-utils/web-apps/commit/d84d9b77bcf3efcec255b6523708dc3330e2d147

I did a mistake moving icon before finishing creation. So basically each time you selected an icon, it was moved to new directory and path was set in desktop entry. Now, only when finishing creation and entry is valid it will be moved and set new path to entry. Imho it should works better now.
